types/webgl2/index.d.ts

```diff
@@ -486,21 +510,23 @@ interface WebGL2RenderingContext extends WebGLRenderingContext {
     /* Reading back pixels */
     // WebGL1:
     readPixels(x: number, y: number, width: number, height: number, format: number, type: number,
-        dstData: ArrayBufferView | null): void;
+        dstData: Int8Array | Int16Array | Int32Array | Uint8Array | Uint16Array | Uint32Array | Uint8ClampedArray |
```

Comment:
Does this method also require a compatibility line for older Typescript versions?

Reply:
In the other cases I noticed the compatibility line was necessary when running `npm run lint webgl2`. Strangely enough it didn't trigger in this case. 

I would suggest still adding it, or WDYT?